verilog/CST/expression_test.cc

```diff
@@ -664,6 +664,33 @@ TEST(ReferenceIsSimpleTest, Simple) {
   }
 }
 
+TEST(ReferenceIsSimpleButInReferenceCallBaseTest, Simple) {
+  const char* kTestCases[] = {
+      // as this will get wrapped in the data declaration for foo(), only a will
+      // be matched
+      "foo(a);",
+  };
+  for (auto code : kTestCases) {
```

Comment:
`for (const auto* code in kTestCases)`

Reply:
The test was based around `foo(` being a kDataDeclaration, but since this got unified to be a function call, it is not there anymore (also I'm explaining the removal of said test).